class-components/src/components/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const NotFound: React.FC = () => {
+  return (
+    <div>
+      <h2>404 - PAGE NOT FOUND</h2>
+    </div>
+  );
+};
+
+export default NotFound;
```

Comment:
Should such a page maybe contain a link to the main page? This way, it doesn't look user-friendly